packages/cache/package.json

```diff
@@ -4,6 +4,18 @@
   "description": "emotion's cache",
   "main": "dist/cache.cjs.js",
   "module": "dist/cache.esm.js",
+  "browser": {
+    "./dist/cache.cjs.js": "./dist/cache.browser.cjs.js",
+    "./dist/cache.esm.js": "./dist/cache.browser.esm.js"
+  },
+  "react-native": {
+    "./dist/cache.cjs.js": "./dist/cache.native.cjs.js",
+    "./dist/cache.esm.js": "./dist/cache.native.esm.js"
+  },
+  "sketch": {
+    "./dist/core.cjs.js": "./dist/core.native.cjs.js",
```

Comment:
shouldnt this be `cache` instead of `core`? 😉

Reply:
Oh ya! My bad.. Updated that